### ISSUE TITLE
Interaction avec asgardMenu

### ISSUE THREAD

**Issue by FERRATON** [MTES-MCT/metadata-postgresql#52]
Le prototype que j'avais réalisé interagissait avec AsgardMenu, ce que ne fait pas la version actuelle de Plume.
Pour cela il faut rechercher la présence d'un Qwidget de nom 'Menu Tree'. C'est le nom historique du plugin Oslandia pour AsgardMenu. 
Si necessaire je peux le changer dans une nouvelle version pour que ce soit par exemple 'Asgard Menu'...
il faut donc ajouter quelque chose comme :
        if iface.mainWindow().findChildren(QWidget, 'Menu Tree') :
            self.asgardTree = iface.mainWindow().findChildren(QWidget, 'Menu Tree')[0] #cf si plusieurs instances de AsgardMenu
            self.asgardTreeView = self.asgardTree.findChild(QTreeView)
            self.asgardTreeView.clicked.connect(self.retrieveCommentAsgard)
et ensuite 'cabler' la fonction retrieveCommentAsgard pour retrouver les paramètres de la source...
...
        self.mProxyModel = self.asgardTreeView.model()
        self.mModel = self.mProxyModel.sourceModel()
        self.item = self.mModel.itemData(self.mProxyModel.mapToSource(index))
        self.layer = QgsVectorLayer(self.item[257].uri, self.item[0], 'postgres') 

a adapter...

**Comment by WREATCHED** [comment]
Oui, c'est dans notre tête et pas un pb technique, mais des priorités.

**Comment by alhyss** [comment]
Merci pour ces infos @FERRATON.

Comme le dit @WREATCHED, nous nous étions limités à deux canaux (explorateur + panneau des couches) pour la première version de Plume, car il ne semblait pas y avoir une très forte demande sur l'ouverture des fiches de métadonnées depuis le panneau d'AsgardMenu.

J'ai ajouté ce sujet à la liste des potentielles fonctionnalités de la v2, pour discussion en sous-groupe Métadonnées (cf. issue #19), comme ça nous ne risquerons pas de l'oublier.

D'une manière générale, je suis bien d'accord sur le fait qu'il est souhaitable de bien coordonner entre eux les outils du GT PostGIS. Néanmoins gérer un canal supplémentaire n'est pas neutre et je pense qu'il nous faut ici prendre le temps de confirmer qu'il y a bien un besoin. Outre les retours que nous aurons des membres du GT, as-tu des éléments sur l'adoption d'AsgardMenu par les services, et sais-tu si les utilisateurs préfèrent la forme panneau (la seule pour laquelle l'affichage des métadonnées serait possible) ou la forme menu ?

**Comment by FERRATON** [comment]
Pas de pb pour la gestion des priorités. C'était juste pour mémoire.
Je n'ai pas de retour sur l'usage de AsgardMenu. Perso je préfère de loin la forme panneau, mais les habitudes des utilisateurs ne sont pas toujours faciles à changer (historique avec Menu GéoIDE,...).
